### ISSUE TITLE
dependencies files not available

### ISSUE THREAD

**Issue by lejarx** [toebR/Own_R_Viz_Projects#1]
Hi could you please share how to retrieve the other files?

For example, I may be interested in visualizing the country of France.
From the below, it's quite difficult to know where and how to get the shapefiles.

Thank you

https://github.com/toebR/Own_R_Viz_Projects/blob/master/swiss_roads_map/roads_map.R
```
#load the geodata that I need for the map (was a bit tweaked in QGis =P)
CH <- st_read(dsn = "swissBOUNDARIES3D_1_3_TLM_LANDESGEBIET.shp")
cantons <- st_read(dsn = "swissBOUNDARIES3D_1_3_TLM_KANTONSGEBIET.shp")
roads <- st_read(dsn = "gis_osm_roads_fclass2.shp")
```

**Comment by toebR** [comment]
Hi there,
thanks for writting this issue. Your are right, I did not share the exact locations becaue this script was focused on the mapping part. I retrieved the Swiss coundaries and cantons from a free product called swissBoundaries3D. link: https://shop.swisstopo.admin.ch/en/products/landscape/boundaries3D

The OSM data I gathered the following way: There is a german OSM-based company that assembles OSM data country-wise in different formats. Upon looking for France, I saw that they dont have it in shp format but in others: https://download.geofabrik.de/europe.html

Hope that helps :)
Beste regards, Tobias
